server/chat-plugins/info.js

```diff
@@ -682,7 +682,21 @@ const commands = {
 		if (!this.runBroadcast()) return;
 		target = target.trim();
 		let modName = target.split(',');
-		let mod = Dex.mod(toId(modName[modName.length - 1])) || Dex;
+		let mod = Dex;
+		/** @type {Format?} */
+		let format = null;
+		if (modName[1] && toId(modName[1]) in Dex.dexes) {
```

Comment:
You can't use `modName[1]` because `/weak` supports multiple types in an argument. Instead, select the last argument with `modName[modName.length - 1]`

Reply:
My bad, should be fixed now.